### PR TITLE
RNMT-2573 Upgrade iOS MobileECT

### PR DESCRIPTION
**Pull request by EiyuuZack**
## Description

Adds support for WKWebView.

## Context

Closes https://outsystemsrd.atlassian.net/browse/RNMT-2573

See the commit messages for more information.

## Type of changes

- [x] Feature (change which adds functionality)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to new feature)
- [ ] Fix (change which fixes an issue)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to fix)
- [ ] Performance (change which improves performance)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to performance improvement)
- [ ] Refactor (non-breaking change that is neither feature, fix nor performance)
- [ ] Style (non-breaking change that only affects formatting and/or white-space)

## Components affected

- [ ] Android
- [x] iOS
- [ ] JavaScript

## Tests
Manual testing

## Screenshots (if appropriate)

appfeedback_wkwebview
Screenshot 2019-08-07 at 11 05 49
Screenshot 2019-08-07 at 10 50 18

## Checklist

- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [ ] Tests have been created
- [x] Code follows code style of this project
- [ ] CHANGELOG.md file is correctly updated
- [ ] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

File: src/ios/OSAppFeedback.m [hunk 1]
```diff
@@ -255,11 +255,9 @@ -(void)handleIsECTAvailable: (CDVInvokedUrlCommand*) command{
 
 
 -(void) initEctController:(NSString*) hostname {
-    UIWebView *webView = (UIWebView*)self.webViewEngine;
-    
     // Mobile ECT Configuration
     self.mobileECTController = [[OSMobileECTController alloc] initWithSuperView:_mobileECTView
-                                                                     andWebView:webView
+                                                                     andWebView:self.webView
```
Comment (luissilvaos) [inline src/ios/OSAppFeedback.m:260]: Nice catch!

**Review by luissilvaos — APPROVED** [review]
🚢

**Review by pmaroco — APPROVED** [review]
it!

**Comment by vmfo22** [comment]
@EiyuuZack these changes can be merged for both branches we currently have available in production?

**Review by vmfo22 — APPROVED** [review]

**Comment by EiyuuZack** [comment]
@vmfo22 I do not see why not. Is there a pressing need to do so before MABS 6?

**Comment by vmfo22** [comment]
@EiyuuZack it's not about pressing. I'm asking this because the branch for O10 is currently used for O10 and O11 (at least in some customers)

**Review by Chuckytuh — APPROVED** [review]

**Pull request merged**